services/ocr/ocr.py

```diff
@@ -499,4 +498,4 @@ def slice_processing(img, last_valid_name:str, raw_students:dict):
     #img = Qr.create("01557898-f61c-11ed-b67e-0242ac120002")
     #img.save("Qr.jpg")
 
-    print(Engine.process(cv2.imread("ocr-service/imgs/img1.jpg"), 22))
+    print(Engine.process(cv2.imread("ocr-service/imgs/img1.jpg"), 22, "127.0.0.1"))
```

Comment:
K čemu se tady přidala ta IP adresa?

Reply:
Když chceš testovat, tak by ti to jinak nefungovalo